src/services/hackerNews.js

```diff
@@ -0,0 +1,51 @@
+import http from '../utils/http';
+import API from '../constants/api';
+
+/**
+ * @param {String} newsType
+ * @param {Number} newsId
+ * Fetch news data from the api.
+ */
+export async function getNews(newsType, newsId = null) {
+  if (newsId === null) {
```

Comment:
this logic to getNews doen't make sense explicitly initializing newsId to null doesn't make sense. what are you trying to accomplish here.